docs/Migrating.md

```diff
@@ -1,10 +1,16 @@
 # Migrating
 
 ## Migrating from ML-Agents toolkit v0.5 to v0.6
+
 ### Important
-* Brains are now Scriptable Objects instead of MonoBehaviors. This will
-  allow you to set Brains into prefabs and use the same brains across 
-  scenes.
+
+* Brains are now Scriptable Objects instead of MonoBehaviors. 
+* You can no longer modify the type of a Brain. If you want to switch
+  between `PlayerBrain` and `LearningBrain` for multiple agents,
+  you will need to assign a new Brain to each agent separately.
+  __Note:__ You can pass the same brain to multiple agents in a scene by 
+leveraging Unity's prefab system or look for all the agents in a scene
+using the search bar of the `Hierarchy` window with the word `Agent`.
 * To update a scene from v0.5 to v0.6, you must:
   *  Remove the `Brain` GameObjects in the scene
```

Comment:
Remove the `Brain` GameObjects in the scene -> Delete all of the `Brain` GameObjects under Academy in the scene.